README.md

```diff
@@ -71,6 +71,13 @@ VALVE functions are provided as values to the `condition` column in the field ta
 
 When referencing the "target column", that is either the `column` from the field table, or the `then column` from the rule table.
 
+There are five types of arguments passed to VALVE functions:
+* **function**: another VALVE function
+* **named argument**: some functions have optional args in the format `arg=value` (e.g., `direct=true` in [under](#under))
+* **regex**: Perl-style regex pattern, always single-line (`s/pattern/[flags]` or `s/pattern/replacement/[flags]`)
```

Comment:
There shouldn't be a leading `s` for the first example, which is just matching, right?

Reply:
I thought we always wanted to include the `s`; even for matching it should be single line?

Right now, the grammar expects there to be an `s/` for all regex patterns, but I can change that.

Reply:
We only want the `s` for substitutions, but it should still resolve the trailing slash ambiguity we discussed:

- matching `/foo/flags`
- substitution `s/foo/bar/flags`

Reply:
To clarify, matching will *never* have an `s`, or will it sometimes have an `s`?

Reply:
Never